csrc/quantization/pt_binding.cpp

```diff
@@ -136,10 +136,12 @@ at::Tensor dequantize(at::Tensor& quantized_data,
                               .layout(at::kStrided)
                               .device(at::kCUDA)
                               .requires_grad(false);
-    const int final_dim_size = num_decompressed_elems(quantized_data, num_bits);
```

Comment:
If we aren't using this `num_decompressed_elems` anymore can we also remove the implementation for it?

Reply:
Fixed it, all tests passed locally now ;